tests/disable/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,90 @@
+exports[`test disable.js 1`] = `
+"function a() {
+  // prettier-ignore-next-line
```

Comment:
Is the test producing a false positive somehow? It looks the the code is expecting "prettier-disable-next-line" rather than "...-ignore-..."